subprojects/docs/src/docs/userguide/upgrading_version_5.adoc

```diff
@@ -75,6 +75,11 @@ The `org.gradle.api.Nullable` annotation type has been removed. Use `javax.annot
 
 The deprecated OSGI plugin has been removed.  There are a number of community-provided OSGI plugins available on the link:https://plugins.gradle.org[Gradle Plugin Portal].
 
+The following breaking changes will appear as deprecation warnings with Gradle 5.6:
```

Comment:
I think this sentence is a bit misleading. These are people upgrading to 6.x reading this right? So they probably don't care what happened in _5.6_ (or earlier?). I would maybe just go with something simple like 
_The following deprecated APIs have been removed:_

Anyway, we can also change this later when we have all the changes in and polish the upgrading guide.

Reply:
I copied this from the last upgrade guide. So yeah, let's polish this later.